coalib/bearlib/__init__.py

```diff
@@ -150,6 +156,7 @@ def wrapping_function(*args, **kwargs):
 
         wrapping_function.__metadata__ = new_metadata
 
+        wrapping_function.__doc__ = func.__doc__
```

Comment:
This changes are needed, so that docstring of any bear can't get lost because of removing the the `@warps` tool

Reply:
okay. In cases like this it's even better to make a comment in the code since that will stay even when your github comment will not :)

Reply:
Why doesn't work `@wraps` again?

Reply:
You want to know the reason why I remove the `@warps` form. If yes then because I want to check whether the `run()` method is using decorator or not, but how I am going to do that, so I removed the `@warp` tool so that I can check whether `decorators` is getting called or not .

Reply:
These name are used accordingly in creating a different cases,